hcsctl/e2e/cdi/cdi_test.go

```diff
@@ -1,4 +1,4 @@
-package tests
+package testCDI
```

Comment:
Actually by convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. It's under e2e directory, so I think it's okay to have cdi as a package name. Did you encounter error message for that?